### DISCUSSION TITLE
bootstrap_combo_nav_js middle button is always disabled

### DISCUSSION THREAD

**Discussion by MSeneadza** [ddnexus/pagy#444] [Q&A] [answered]
I'm wondering if it's intentional for the middle button in the bootstrap_combo_nav_js to be disabled & non-editable by the user.  Here's a screenshot of what I'm seeing in my app with Bootstrap 5.2.3:

<img width="497" alt="Screen Shot 2023-01-12 at 11 28 39 AM" src="https://user-images.githubusercontent.com/1416050/212126688-7d659a99-284a-41f9-b15a-c142b7da2660.png">

I see from the screenshot here https://ddnexus.github.io/pagy/docs/extras/bootstrap/ that the page number field appears to be editable.  It makes sense to me that the user should be able to enter a page number there and then hit ENTER to navigate to the entered page.  

I modified this behavior in another app of mine by making my own version of the pagy_bootstrap_combo_nav_js method and simply removing the 'disabled' class from that input.  I can certainly do that again but I just thought I'd ask if this is the intended behavior.

**Comment by benkoshy** [comment]
It's a known issue. 

compatibility is broken across bootstrap versions.

**Reply by MSeneadza** [reply]
OK, thanks for the reply.  I'll just change it locally again.

**Comment by ddnexus** [comment]
@benkoshy Then we should reopen #381

**Reply by benkoshy** [reply]
I think we should either drop support for earlier versions of bootstrap,  or have x2 extras catering for different versions? What do you think?

**Reply by ddnexus** [reply]
Yes, absolutely. 

I think we should remove the `disabled` all together and use a bootstrap official class that colors the item in a decent way. If we can do that, it would probably have no backward compatible issues, otherwise limit the support as you suggested.

**Reply by ddnexus** [reply]
@benkoshy @MSeneadza 

We could just edit this line
https://github.com/ddnexus/pagy/blob/3f0aa6d8208948f9e0b516cc8794b0b4bfd03b35/lib/pagy/extras/bootstrap.rb#L64

changing `btn-primary disabled` to `btn-secondary`, and there should be no backward compatibility issues.

Could anyone post a screenshot of that applied change?

**Reply by ddnexus** [reply]
It works. I will commit it. 

![bootstrap-secondary](https://user-images.githubusercontent.com/100721/212300431-8fc642c3-4c10-4650-89cd-99f53e974ff6.png)

I will also restore the normal input text color (that in the screenshot is primary)

**Reply by ddnexus** [reply]
https://github.com/ddnexus/pagy/actions/runs/3910630563

**Reply by benkoshy** [reply]
Looks like an excellent solution.  I had tried to retain the color but had failed.

**Reply by MSeneadza** [reply]
Seems like a great solution to me.  Thanks!

**Comment by ddnexus** [comment, marked as answer]
Resolved in version 6.0.1
